plugins/woocommerce-blocks/tests/e2e/tests/checkout/checkout-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -675,10 +674,6 @@ test.describe( 'Billing Address Form', () => {
 							page.locator( '#billing-state input' )
 						).toHaveValue( value );
 						break;
-					default:
-						await expect(
-							page.locator( `#billing-${ key }` )
-						).toHaveValue( value );
```

Comment:
Did we remove it because of the unrelated company field?

Reply:
Yes!